crates/bitwarden-exporters/src/cxf/identity.rs

```diff
@@ -0,0 +1,461 @@
+
+    let (first_name, last_name) = split_name(&passport.full_name);
+
+    let identity = Identity {
+        first_name,
+        last_name,
+        // Map nationalIdentificationNumber to ssn as closest available field
+        ssn: passport
+            .national_identification_number
+            .as_ref()
+            .map(|n| n.value.0.clone()),
+        passport_number: passport.passport_number.as_ref().map(|p| p.value.0.clone()),
+        ..Default::default()
+    };
+
+    // Create custom fields for unmapped data according to CXF mapping document
+    let custom_fields = [
+        passport
+            .issuing_country
+            .as_ref()
+            .map(|issuing_country| create_field("Issuing Country", issuing_country)),
+        passport
+            .nationality
+            .as_ref()
+            .map(|nationality| create_field("Nationality", nationality)),
+        passport
+            .birth_date
+            .as_ref()
+            .map(|birth_date| create_field("Birth Date", birth_date)),
+        passport
+            .birth_place
+            .as_ref()
+            .map(|birth_place| create_field("Birth Place", birth_place)),
+        passport.sex.as_ref().map(|sex| create_field("Sex", sex)),
+        passport
+            .issue_date
+            .as_ref()
+            .map(|issue_date| create_field("Issue Date", issue_date)),
+        passport
+            .expiry_date
+            .as_ref()
+            .map(|expiry_date| create_field("Expiry Date", expiry_date)),
+        passport
+            .issuing_authority
+            .as_ref()
+            .map(|issuing_authority| create_field("Issuing Authority", issuing_authority)),
+        passport
+            .passport_type
+            .as_ref()
+            .map(|passport_type| create_field("Passport Type", passport_type)),
+    ]
+    .into_iter()
+    .flatten()
+    .collect();
+
+    (identity, custom_fields)
+}
+
+/// Convert person name credentials to Identity and custom fields
+/// According to CXF mapping:
+/// - title: EditableField<"string"> → Identity::title
+/// - given: EditableField<"string"> → Identity::first_name
+/// - given2: EditableField<"string"> → Identity::middle_name
+/// - surname: EditableField<"string"> → Identity::last_name
+/// - surnamePrefix + surname + surname2: combine for complete last name
+/// - credentials: EditableField<"string"> → Identity::company (as professional credentials)
+/// - Other fields → CustomFields
+pub(super) fn person_name_to_identity(
+    person_name: &PersonNameCredential,
+) -> (Identity, Vec<Field>) {
+    // Construct complete last name from surnamePrefix, surname, and surname2
+    let last_name = [
+        person_name.surname_prefix.as_ref(),
+        person_name.surname.as_ref(),
+        person_name.surname2.as_ref(),
+    ]
+    .into_iter()
+    .flatten()
+    .map(|field| field.value.0.clone())
+    .collect::<Vec<_>>()
+    .into_iter()
+    .reduce(|acc, part| format!("{} {}", acc, part));
```

Comment:
This will be a warning in the latest version of rust:
```suggestion
    .reduce(|acc, part| format!("{acc} {part}"));
```